complex/cargo/list/src/items.rs

```diff
@@ -32,6 +33,15 @@ pub struct Item {
     pub labels: Vec<Label>,
 }
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct ItemJNA {
```

Comment:
This is something I have been considering doing for all objects we pass over the boundary - converting them from complex Rust types to C structs. This would also make handling the objects in Swift much easier.